### ISSUE TITLE
[Android] Retroarch font size is too small

### ISSUE THREAD

**Issue by FeRcHuLeS** [libretro/RetroArch#3977]
The latest version 1.3.6 I installed has the font too small, and is imposible to use because of the screen size,unless you've got a controller, the phone is the Xperia X10a using android 2,3,3,

**Comment by fr500** [comment]
It's perfectly fine on my device show a screenshot I guess

**Comment by FeRcHuLeS** [comment]
Here a screenshot:
[https://drive.google.com/open?id=0BzLzYSdAsr2hdkp1UjVLTXZCSFU](url)
and where to find the nightlies? thanks.

**Comment by lasers** [comment]
Nightlies -- https://buildbot.libretro.com/nightly/ 

Try: Settings --> Menu --> DPI Override --> (Go Higher)

**Comment by FeRcHuLeS** [comment]
Thanks, That worked but was hard to configure it, I had to use a magnifying glass lol.

**Comment by lasers** [comment]
Close this?

**Comment by fr500** [comment]
No, the issue persists, DPI detection is not working on that android version.

**Comment by zoltanvb** [comment]
I think this can be closed - Android 2.3.3 / API 10 is not supported since long by current RetroArch builds, and no reports of similar problems in newer OSes.
